samples/SortBenchmark/DiskHelper.fs

```diff
@@ -291,3 +299,85 @@ type DiskHelper(records:int64) =
                 x.sorteddataSP.Save() 
                 b <- true
                 x.sorteddataSP.lock := 0
+
+
+
+[<AllowNullLiteral>]
```

Comment:
I agree. Actually, this type may be removed later since we changed the implementation.